lib/default.nix

```diff
@@ -166,16 +166,23 @@ let
             };
           };
 
-          loadNixDarwin = path: {
-            class = "nix-darwin";
-            value = inputs.nix-darwin.lib.darwinSystem {
-              modules = [
-                perSystemModule
-                path
-              ];
-              inherit specialArgs;
+          loadNixDarwin =
+            path:
+            let
+              nix-darwin =
+                inputs.nix-darwin
+                  or (throw ''error: ${path} depends on nix-darwin. To fix this, add `inputs.nix-darwin.url = "github:Lnl7/nix-darwin";` to your flake'');
```

Comment:
When the error is displayed by the CLI, it will already be prefixed with `error: `